google/cloud/internal/oauth2_service_account_credentials.h

```diff
@@ -30,6 +30,11 @@ namespace google {
 namespace cloud {
 namespace oauth2_internal {
 GOOGLE_CLOUD_CPP_INLINE_NAMESPACE_BEGIN
+namespace {
+
+auto constexpr kP12PrivateKeyIdMarker = "--unknown--";
+
+}  // namespace
```

Comment:
How about I make it `static constexpr` and remove the anonymous namespace?

Reply:
Candidly, I don't know if that will work.  We may run into this:

https://cmake.org/cmake/help/latest/prop_tgt/WINDOWS_EXPORT_ALL_SYMBOLS.html#:~:text=For%20global%20data%20symbols%2C%20__declspec(dllimport)%20must%20still%20be%20used%20when%20compiling%20against%20the%20code%20in%20the%20.dll.

Then again, you may know better and maybe a `constexpr` like this would be harmless.  If you know for certain it would work, then by all means let's do that.  If there is any doubt, it does not seem worth the risk: this code runs like once in a program, and I doubt the inline function call will have any measurable effect in performance or object size.

Reply:
Changed to your recommendation out of an abundance of caution.